README.md

```diff
@@ -49,32 +49,32 @@ The default duration is *60 seconds*, and the default interval matches it. So gP
 
 ### Java profiling options
 
-* `--no-java`: Disable profilers for Java.
+* `--no-java` or `--java-mode disabled`: Disable profilers for Java.
 
 ### Python profiling options
 * `--no-python`: Alias of `--python-mode none`.
 * `--python-mode`: Controls which profiler is used for Python.
     * `auto` - (default) try with PyPerf (eBPF), fall back to py-spy.
     * `pyperf` - Use PyPerf with no py-spy fallback.
     * `pyspy` - Use py-spy.
-    * `none` - Disable profilers for Python.
```

Comment:
This breaks compatibility, and we have users that already use `none`.

I suggest: remove `none` from here, but maintain compatibility in code

Reply:
Yep, supporting `none` behind the scenes